test/integration/suites/agent-cli/05-check-valid-config

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+RETRIES=10
+AGENT_FOUND=0
+VALID_CONFIG=0
+INVALID_CONFIG=0
+
+
+for ((m=1;m<=$RETRIES;m++)); do
```

Comment:
since step 4 will validate that agent is up, I think the same validation in all test cases is not required